src/main/kotlin/no/nav/syfo/dialogmote/api/domain/NewDialogmoteDTO.kt

```diff
@@ -15,13 +15,31 @@ data class NewDialogmoteDTO(
 data class NewDialogmotedeltakerArbeidstakerDTO(
     val personIdent: String,
     val fritekstInnkalling: String?,
+    val innkalling: List<DocumentComponentDTO>,
 )
 
 data class NewDialogmotedeltakerArbeidsgiverDTO(
     val virksomhetsnummer: String,
     val fritekstInnkalling: String?,
+    val innkalling: List<DocumentComponentDTO>,
 )
 
+data class DocumentComponentDTO(
```

Comment:
Synes vi bør finne et passe generisk begrep for dette som kan benyttes både for innkalling og referat (til både arbeidstaker og arbeidsgiver).

Reply:
Enig!